mb_netmgmt/__main__.py

```diff
@@ -104,6 +104,14 @@ def get_to(self):
             proxy = self.get_proxy(stubs[-1])
             if not proxy:
                 proxy = self.get_proxy(stubs[0])
+            return proxy
+        except (IndexError, AttributeError, requests.RequestException):
+            return None
```

Comment:
do we really want to pretend that no proxy is configured and continue with the flow in case of those errors?